index.html

```diff
@@ -2,11 +2,13 @@
 <html lang="en">
 
 <head>
```

Comment:
Please make sure the README has instructions to set the project up locally.

Reply:
This issue is not properly resolved

Reply:
This issue is not properly resolved